website/src/app/kb/use-cases/host-access/readme.mdx

```diff
@@ -0,0 +1,71 @@
+import SupportOptions from "@/components/SupportOptions";
+import Alert from "@/components/DocsAlert";
+import Image from "next/image";
+import Link from "next/link";
+import PlanBadge from "@/components/PlanBadge";
+
+<PlanBadge plans={["starter", "team", "enterprise"]}>
+
+# Access a Remote Host
+
+</PlanBadge>
+
+In this guide, we'll be using Firezone to secure access to a host based on the
+host's private IP address.
+
+This is useful when you have a host that may be behind a firewall that you want
+to keep secure, but still need to access it from external networks like the
+internet.
+
+After completing this guide, you'll be able to access the host using its private
+IP address and any services running on it, like SSH or RDP.
+
+## Prerequisites
+
+- A Site that contains the host you want to secure access to.
```

Comment:
Actually the host isn't in that Site until it's created as a Resource in Step 1.

Like, it's logically in the Site, from the customer saying "We have this building, that's a site", but it's not in the Firezone capital-S Site until you're following the instructions.

Maybe "A Site that _will_ contain the host..."